lib/scidata/squad.ex

```diff
@@ -0,0 +1,189 @@
+defmodule Scidata.Squad do
+  @moduledoc """
+  Module for downloading the [SQuAD1.1 dataset](https://rajpurkar.github.io/SQuAD-explorer).
+  """
+
+  require Scidata.Utils
+  alias Scidata.Utils
+
+  @base_url "https://rajpurkar.github.io/SQuAD-explorer/dataset/"
+  @train_dataset_file "train-v1.1.json"
+  @test_dataset_file "dev-v1.1.json"
+
+  @doc """
+  Downloads the SQuAD dataset.
+
+  ## Examples
+
+      iex> Scidata.Squad.download()
+      %{
+        [
+          %{
+            "paragraphs" => [
+              %{
+                "context" => "In many cities along the North American...",
+                "qas" => [
+                  %{
+                    "answers" => [%{"answer_start" => 324, "text" => "hundreds"}],
+                    "id" => "56d8a0ddbfea0914004b7706",
+                    "question" => "How many people protested on the San Francisco torch route?"
+                  },
+                  ...
+                ]
+              }
+              ...
+            ]
+          }
+        ]
+      }
```

Comment:
```suggestion
      [
        %{
          "paragraphs" => [
            %{
              "context" => "Architecturally, the school has a...",
              "qas" => [
                %{
                  "answers" => [%{"answer_start" => 515, "text" => "Saint Bernadette Soubirous"}],
                  "id" => "5733be284776f41900661182",
                  "question" => "To whom did the..."
                }, ...
              ]
            }
          ],
          "title" => "University_of_Notre_Dame"
        }, ...
      ]
```

Let's preview the return list with the first example.